### DISCUSSION TITLE
plus 7B的拒答有点严重啊

### DISCUSSION THREAD

**Discussion by xpeng** [ymcui/Chinese-LLaMA-Alpaca#253] [General]
非常感谢模型作者的工作，给我们带来了更新的中文模型。

plus 7b出来后，我仍然以角色模拟（Generative Agent）场景测试了最新出的plus 7B，发现这个版本拒答很严重啊（拒答评测分数很高），只有重新调节了指令prompt，做了一些语言陷阱才算通过😁。

想问下大家，这里有没有一种伦理参数可以调节伦理拒答程度的啊？否则这种“焊死”了的模型，在做角色模拟类实验的时候就比较困难了

**Comment by ymcui** [comment]
训练数据中没有特别加入”角色扮演“类的指令数据，所以拒答方面主要还是以理解通常的指令为目的。
可以考虑在我们的模型上进一步使用此类指令数据精调。

P.S. 模型中没有”伦理参数“供调节。
